.github/workflows/scheduled-test.yml

```diff
@@ -0,0 +1,74 @@
+name: Run tests
+on:
+  schedule:
+    - cron: "0 8 * * *"
+  workflow_dispatch: {}
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  PULUMI_TEST_OWNER: "moolumi"
+  PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
+  AWS_REGION: "us-west-2"
+  PULUMI_API: https://api.pulumi-staging.io
+
+jobs:
+  test:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      matrix:
+        platform:
+          - ubuntu-latest
+        go-version:
+          - 1.18.x
```

Comment:
I noticed we have 1.19 in other workflows. Should we bump it to be consistent?

Reply:
We should keep to the versions Pulumi supports, each of which are linked from [our language pages](https://www.pulumi.com/docs/languages-sdks/). As of today, that is:

* Node 18
* Python 3.8
* Go 1.20
* .NET 6
* Java 11

I've updated all programs and workflows accordingly. Thanks for the callout!

Reply:
Just a general note regarding Python: 3.8 will be going full EoL (no more security updates) sometime in 2024, so it may be a good idea to bump that to something newer.